css/base.css

```
@@ -0,0 +1,113 @@
+html {
+  background: var(--bg);
+  color: var(--text);
+  color-scheme: light;
+}
+
+body {
+  font-family: var(--serif);
+  font-size: var(--normal);
+  font-size-adjust: from-font;
+  line-height: 1.3;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: var(--sans-serif);
+}
+
+h1 {
+  font-size: var(--h1);
+}
+
+h2 {
+  font-size: var(--h2, var(--large));
+}
+
+ul,
+ol,
+p {
+  margin-block: 0.5lh;
+}
+
+ul,
+ol {
+  margin-inline: 2ch;
+  padding-inline: unset;
+
+  li & {
+    margin-block: 0;
+  }
+}
+
+:focus {
```

```suggestion
:focus-visible {
```

I think this is pretty safe at this point.